modules/repofiles/update.go

```diff
@@ -5,6 +5,7 @@
 package repofiles
 
 import (
+	"code.gitea.io/gitea/modules/log"
```

Comment:
Need to move this down to join the rest of the gitea imports

Reply:
Fixed.